readme.md

```diff
@@ -14,11 +14,11 @@
 
 ##### Installation
 
-The `R` package is presently under development and therefore not currently listed on [`CRAN`](https://cran.r-project.org/). For the meantime, please download the `R` package file associated with the [latest release](https://github.com/psych-ds/validator/releases/latest).
-
-In `RStudio`, please then select `Install Packages ...` from the `Tools` menu. Next, please choose `Package Archive File` in the `Install from` dropdown, and select the file you downloaded in the last step. After confirming your selection, the installation should proceed automatically.
-
-Alternatively, you can install the package from the `R` command line with the command `install.packages("~/path/to/download/psychds_x.y.z.tar.gz", repos=NULL, type="source")`.
+The `R` package is presently under development and therefore not currently listed on [`CRAN`](https://cran.r-project.org/). In the meantime, you can install the latest version of the package from the R command line with the commands:
+```
+install.packages(devtools)  # Can be omitted if devtools is already installed
+devtools::install_url(url = "https://github.com/psych-ds/validator-prototype/files/3638496/psychds_0.1.0.tar.gz")
```

Comment:
The URL here is only linked from one of the comments, and very much not official 😅. I've put together a [proper release](https://github.com/psych-ds/validator-prototype/releases) now, and you could point to the attached package file, however I would much prefer if we found some way to teach users to go to the release page and pick the latest file, because (from experience) continuously updating the link will be an error-prone process.

We're going to get to CRAN eventually, which will fix installation issues, so this isn't a workflow that many users will deal with in the future, but for the time being it would be great to have a clear documentation that doesn't require frequent updating.

Reply:
Ah, one alternative would be to make a template for the release message that includes the installation command specific to that release, but I guess it would be nice to have a generally applicable set of instructions in the readme in addition to that.

Reply:
Are you planning to schedule many releases before putting the package on CRAN? If not, the inconvenience of manual updating should be weighed against the inconvenience of the user having to figure out where to navigate to find the latest version of the package. This is where I ran into problems in the first place. IMO as a user, a simple "download here" button on the validator front page is a UX improvement that outweighs the risk of not downloading the latest version of the package. 

However, this is all just temporary and I don't think it will matter much in the long run. I will keep the direct links in my pull request. If you think that is a bad idea, just override my suggestions and write what you think is best. No need to ask for permission :)

Reply:
> Are you planning to schedule many releases before putting the package on CRAN?
That's a great point: I don't know TBH, I think it's probably going to be up to Melissa and you folks to decide when things are ready. From what I understand, Melissa would rather wait until the spec is semi-stable, and I couldn't tell when that will be.

> IMO as a user, a simple "download here" button on the validator front page is a UX improvement that outweighs the risk of not downloading the latest version of the package.
Excellent point, I'm convinced! Can you help me try to keep the link updated? Or, thinking about it, would it make sense to have a note in the readme that says "this link might not be up-to-date, please check the release page", so that folks who actually read the text will get to use a more recent version?

> I will keep the direct links in my pull request.
👍 I see your reasoning, totally on board with your suggestion. Thanks a lot!